src/libs/actions/PersonalDetails.js

```diff
@@ -81,6 +82,24 @@ function extractFirstAndLastNameFromAvailableDetails({
     };
 }
 
+/**
+ * Convert country names to their respective ISO codes obtained from the backend
+ * This is for backward compatibility of stored data before E/App#15507
+ * @param {String} countryName
+ * @returns {String}
+ */
+function getCountryISO(countryName) {
+    if (countryName === '' || countryName.length === 2) {
```

Comment:
Why not use `_.isEmpty`?

```suggestion
    if (_.isEmpty(countryName) || countryName.length === 2) {
```